ArduCopter/mode.h

```diff
@@ -405,6 +405,11 @@ class ModeAuto : public Mode {
 
 private:
 
+    enum class Options : int32_t {
+        AllowArming    = (1 << 0U),
```

Comment:
By tradition these labels are always upper-case.

Reply:
Accepted on the call as is for readability. Party's were had.